web/src/engine/osk/src/input/gestures/specsForLayout.ts

```diff
@@ -643,7 +643,8 @@ export function longpressModel(params: GestureParams, allowShortcut: boolean, al
         },
         endOnResolve: false
       }, {
-        model: instantContactRejectionModel()
+        model: instantContactRejectionModel(),
+        resetOnInstantFulfill: true
```

Comment:
The actual fix, on the 'longpress' model.  It didn't reset the match-state of any auto-cancelling touches.